src/sentry/services/hybrid_cloud/auth/serial.py

```diff
@@ -87,6 +87,7 @@ def serialize_api_token(at: ApiToken) -> RpcApiToken:
         organization_id=at.organization_id,
         application_is_active=at.application_id is None or at.application.is_active,
         token=at.token,
+        hashed_token=at.hashed_token,
```

Comment:
The timing on this could bite us. We could end up serializing entities that can't be unserialized on the receiving side.  If you wanted to be extra cautious we would need to add the field to `RpcApiToken` deploy that, and then separately start hydrating the field.